README.md

```diff
@@ -1,125 +1,118 @@
+nvm install latest
+```
 
-```fish
-set -q XDG_CONFIG_HOME; or set XDG_CONFIG_HOME ~/.config
+Install the latest [LTS](https://github.com/nodejs/Release) (long-term support) Node release.
 
-for i in conf.d functions completions
-  curl https://git.io/$i.nvm.fish --create-dirs -sLo $XDG_CONFIG_HOME/fish/$i/nvm.fish
-end
+```console
+nvm install lts
 ```
 
-To uninstall nvm, just run:
+Install an older LTS release by codename.
 
-```
-rm -f $XDG_CONFIG_HOME/fish/{conf.d,functions,completions}/nvm.fish && emit nvm_uninstall
+```console
+nvm install carbon
 ```
 
-</details>
+> Installs `8.16.2`, the latest release of the Carbon LTS line.
 
-## Quickstart
-
-Download and switch to the latest Node.js release.
+Or install a specific version of Node.
 
 ```console
-nvm use latest
+nvm install v12.9.1
 ```
 
-> **Note:** This downloads the latest Node.js release tarball from the [official mirror](https://nodejs.org/dist), extracts it to <code>[\$XDG_CONFIG_HOME](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#variables)/nvm</code> and modifies your `$PATH` so it can be used immediately. Learn more about the Node.js release schedule [here](https://github.com/nodejs/Release).
+> Supports full or partial version numbers, starting with an optional "v".
 
-Download and switch to the latest LTS (long-term support) Node.js release.
+The `nvm install` command activates the specified Node version only in the current environment. If you want to set the default version for new shells use:
 
-```console
-nvm use lts
+```fish
+set -U nvm_default_version v12.9.1
 ```
 
-You can create a `.nvmrc` file in the root of your project (or any parent directory) and run `nvm` to use the version in it. `nvm` will try to find the nearest `.nvmrc` file, traversing the directory tree from the current working directory upwards.
+Activate a version you've already installed.
 
 ```console
-node -v > .nvmrc
-nvm
+nvm use lts
 ```
 
-Run `nvm` in any subdirectory of a directory with an `.nvmrc` file to switch to the version from that file. Similarly, running `nvm use <version>` updates that `.nvmrc` file with the specified version.
+List which versions you have installed (including the system Node if there is one).
 
 ```console
-├── README.md
-├── dist
-├── node_modules
-├── package.json
-└── src
-    └── index.js
+$ nvm list
+     system  
+    v8.17.0 lts/carbon 
+    v12.9.1  
+ ▶ v14.15.1 lts/fermium 
+    v15.3.0 latest
 ```
 
+Or list all the Node versions available to install.
+
 ```console
-echo lts >.nvmrc
-cd src
-nvm
-node -v
-v10.15.1
+$ nvm list-remote
+   ...
+   v14.13.1  
+   v14.14.0  
+   v14.15.0 lts/fermium 
+ ▶ v14.15.1 lts/fermium ✓
+    v15.0.0  
+    v15.0.1  
+    v15.1.0  
+    v15.2.0  
+    v15.2.1  
+    v15.3.0 latest ✓
 ```
 
-### Listing versions
-
-List all the supported Node.js versions you can download and switch to.
+Want to remove a Node version? You can do that too.
 
 ```console
-nvm ls
+nvm remove v12.9.1
```

Comment:
This should be `nvm uninstall`, no? Most/all package managers use `uninstall`. E.g. nvm-sh, apt, helm etc.

Reply:
Do they? It's more mixed than not.

```
pacman -Rs 
dnf remove 
apt remove 
zypper remove 
zypper rm 
yarn remove
npm remove # or uninstall
brew remove # or uninstall
pip uninstall
gem uninstall
helm uninstall 
helm delete
```

> By the way, `uninstall` also works, but it's not tab-completed or mentioned in the docs.

Reply:
The ones in that list that don't offer `uninstall` are yarn, apt, pacman, and zypper/dnf which I've never heard of - given that npm (and nvm), which are the most relevant for the ecosystem in question, use `uninstall`, that seems like the best example to follow.

Reply:
Although not an explicit goal of this project, more compatibility with the original nvm is always a good thing, especially for non-interactive use, which is why we support both `uninstall` and `remove`. I'll ponder this a bit more. Thanks.

Reply:
I know e.g. helm moved to recommend/default `helm uninstall` in v3. (though others are supported for compability). I made the comment mostly because that was how nvm-sh did it, as @ljharb pointed out. I also feel "uninstall" is a better term, purely subjectively 😆 Your decision of course 👍